pandora-server-directory/src/config.ts

```diff
@@ -109,6 +109,15 @@ export const EnvParser = CreateEnvParser({
 	HCAPTCHA_SITE_KEY: z.string().default(''),
 
 	//#endregion
+
+	//#region Account Security
+
+	/** Time window for login attempts */
+	LOGIN_ATTEMPT_WINDOW: EnvTimestamp().default('15m'),
+	/** Max failed login attempts before requiring a captcha */
+	LOGIN_ATTEMPT_LIMIT: z.number().int().positive().default(30),
```

Comment:
Directory's `.env.template` file should be updated with these values.

Reply:
hmm actually I am not quite sure how much we need the env template
.env is copied only once so no further changes are reflected
if we change any value here users will have an override for the old default
I feel it it would be cleaner if users only ever override what they really want to override

(in any case this PR is not about this so I just added this to for now)